examples/wgpu.rs

```diff
@@ -231,6 +215,7 @@ impl WindowHandler for Wgpu {
             alpha_mode: wgpu::CompositeAlphaMode::Auto,
             width: self.width,
             height: self.height,
+            desired_maximum_frame_latency: 2,
```

Comment:
Where is this field being used?

Reply:
This is a field of `wgpu::SurfaceConfiguration`. That used to be hard-coded to 2, but is now configurable.

Reply:
Oh okay so the web UI probably didn't show the struct owning the field.